src/PerformanceBenchmarkingUtilities.cxx

```diff
@@ -86,7 +87,12 @@ std::string PerfDateStamp()
 {
   std::time_t t = std::time(nullptr);
   char mbstr[100];
+#if defined(_MSC_VER)
```

Comment:
The usual way is to add `_CRT_SECURE_NO_WARNINGS` define, instead of complying with Microsoft's `printf_s` etc functions. We can make the exception this time if you don't want to change the PR.

Reply:
Thanks for the tip. I add the suppression...